app/models/course/forum/topic.rb

```diff
@@ -44,10 +44,8 @@ class Course::Forum::Topic < ApplicationRecord
   # @!attribute [r] post_count
   #   The number of posts in this topic.
   calculated :post_count, (lambda do
-    Course::Discussion::Topic.joins(:posts).
-      where('actable_id = course_forum_topics.id').
-      where(actable_type: Course::Forum::Topic.name).
-      select("count('*')")
```

Comment:
Try not to commit style changes for upgrades like these, it'll end up causing more clutter when reviewing.

Reply:
Done